open-api/rest-catalog-open-api.yaml

```diff
@@ -58,7 +58,9 @@ servers:
         default: ""
 # All routes are currently configured using an Authorization header.
 security:
+  - OAuth2: [catalog]
```

Comment:
I added just one scope for all catalog and config access. Alternatively, we could separate this into read and read/modify scopes to allow read-only access.

Reply:
I think the one scope is fine for now.